src/dtx/dtx_common.c

```diff
@@ -1468,37 +1468,28 @@ dtx_reindex_ult(void *arg)
 	struct ds_cont_child		*cont	= arg;
 	struct dss_module_info		*dmi	= dss_get_module_info();
 	uint64_t			 hint	= 0;
-	int				 rc;
+	int				 rc	= 0;
 
-	D_DEBUG(DB_ANY, DF_CONT": starting DTX reindex ULT on xstream %d\n",
-		DP_CONT(NULL, cont->sc_uuid), dmi->dmi_tgt_id);
+	D_INFO(DF_CONT": starting DTX reindex ULT on xstream %d, ver %u\n",
```

Comment:
I assume this is rare enough that it's not going to fill up the log?

Reply:
Yes, with this patch, it can happen under the following two cases:
1. open container.
2. pool map refresh.